db/migrate/20180730134426_add_port_status_to_physical_network_ports.rb

```diff
@@ -0,0 +1,5 @@
+class AddPortStatusToPhysicalNetworkPorts < ActiveRecord::Migration[5.0]
+  def change
+    add_column :physical_network_ports, :port_status, :string
```

Comment:
This looks unrelated to the rest of this PR

Reply:
@agrare It is, kinda. Because Canister will have some physical ports in it, and I didn't found any reference to this, so I created it. If you want to, I can put it in another PR.

Reply:
Yeah lets do this in a new PR since port_status can apply to anything not just canisters.  So a canister will have hardware -> guest_devices -> physical_network_ports?

Reply:
> So a canister will have hardware -> guest_devices -> physical_network_ports?

@agrare Exactly, this relationship it's currently on the core PR.

Reply:
> Yeah lets do this in a new PR since port_status can apply to anything not just canisters.

@agrare I've done it and put as dependency on this PR, could you take a look there?